src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx

```diff
@@ -623,48 +607,107 @@ For accounts on New Relic One pricing, learn more about [user-related billing ca
+A user group allows managing multiple users at the same time. Your New Relic users are assigned to a group and that group is granted access to specific roles and specific accounts.
+
+Full users can be added to one of two default groups:
+
+* **User**: This group allows a user to use and configure monitoring/analysis features but **not** perform account-related tasks like managing billing or users.
+* **Admin**: This group has all [standard roles](#standard-roles). 
+
+Pro and Enterprise tier organizations can create custom groups, and control access to roles and accounts using [access grants](/docs/accounts/accounts-billing/new-relic-one-pricing-users/organization-level-user-management-roles-groups-access-grants). 
+
+To change the group a user is in, use the [**User management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#where).
 
-## Default groups: Admin and User [#groups]
+
+## Relationship between user type, roles, and groups [#type-role-group]
 
 <Callout variant="important">
   This applies to users on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/transition-guide-new-pricing-user-changes#user-models). For users on our original user model, see [Original user docs](/docs/accounts/original-accounts-billing/original-users-roles).
 </Callout>
 
-A user group allows managing multiple users at the same time. Your New Relic users are assigned to a group and that group is granted access to specific roles and specific accounts.
+Here's a table explaining how [user type](#user-type) (basic vs full user), roles, and groups relate to each other: 
 
-Full users can be added to one of two default groups:
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "100px" }}>
 
-* **User**: This group allows a user to use and configure monitoring/analysis features but **not** perform account-related tasks like managing billing or users.
-* **Admin**: This group has all roles and capabilities. 
+      </th>
 
-See the [roles these default groups have](#group-roles).
+      <th style={{ width: "250px" }}>
+        Full user
+      </th>
 
-For Pro and Enterprise tiers, you can control access to roles and accounts using [access grants](/docs/accounts/accounts-billing/new-relic-one-pricing-users/organization-level-user-management-roles-groups-access-grants). 
+      <th style={{ width: "250px" }}>
+        Basic user
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        **Group**
+      </td>
+
+      <td>
+        Full users can be assigned to [default groups](#groups) (**User** and **Admin**) or custom groups. 
+      </td>
+
+      <td>
+        Basic users can’t be assigned to default groups. They can be assigned to custom groups, but note that a basic user always has basic user permissions, no more no less, even when in a custom group. 
```

Comment:
I like the "no more no less" but is it possible to have less permissions than a basic user?

Reply:
it is possible but only (nonintuitively) if you're a full user who's been specifically assigned a group with less permissions than a basic user. Does that make sense? So full users can theoretically have less permissions than a basic user. A basic user's permissions never change.